src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-detail.component.ts

```diff
@@ -53,6 +68,12 @@ export class FileUploadExerciseDetailComponent implements OnInit, OnDestroy {
                 },
                 (res: HttpErrorResponse) => this.onError(res),
             );
+        this.statisticsService.getExerciseStatistics(exerciseId).subscribe((statistics: ExerciseManagementStatisticsDto) => {
+            this.doughnutStats = statistics;
+            this.participationsInPercent = statistics.numberOfStudentsInCourse > 0 ? round((statistics.numberOfParticipations / statistics.numberOfStudentsInCourse) * 100, 1) : 0;
+            this.questionsAnsweredInPercent = statistics.numberOfQuestions > 0 ? round((statistics.numberOfAnsweredQuestions / statistics.numberOfQuestions) * 100, 1) : 0;
+            this.absoluteAveragePoints = round((statistics.averageScoreOfExercise * statistics.maxPointsOfExercise) / 100, 1);
+        });
```

Comment:
You are doing the same operation whenever you call this service method, maybe you should return them from the service as well.